docs/io/fits/usage/table.rst

```diff
@@ -46,7 +46,7 @@ The underlying data structure used for FITS tables is a class called
 :class:`FITS_rec` which is a specialized subclass of `numpy.recarray`.  A
 :class:`FITS_rec` can be instantiated directly using the same initialization
 format presented for plain recarrays as in the example above.  One may also
-instantiate a new :class:`FITS_rec` from a list of PyFITS `Column` objects
+instantiate a new :class:`FITS_rec` from a list of Astropy `Column` objects
```

Comment:
Use `astropy.io.fits` explicitly here, so user does not get confused with a better known `Column` class in `astropy.table`.